plugins/analytics/public/dockerhub.py

```diff
@@ -77,40 +80,11 @@ def image_full_details(image):
         return image_metadata
 
 
-class DockerImageInspect(task.OneShotTask):
-    """DockerImageInspect analytics queries docker hub to get more
-    details related to docker_image observable.
-
-    This analytics adds several information as context to docker_image:
-    * Image stats: provide details about pulls, registered, updated
-    * Image details: provide information related to image tags including layers information and instructions
-    * User details: provides details about the user owning the image.
-
-    This analytics also creates new observables:
-    * generic observable with tag type:user_account to link to a user
-    * sha256 including image's digest, layers' digests and added files' digests
-    """
-
-    _defaults = {
-        "name": "DockerImageMetadata",
-        "description": "Fetch metadata from docker hub for a docker image",
-    }
-
-    acts_on: list[ObservableType] = [
-        ObservableType.docker_image,
-    ]
-
+class DockerHubObservables:
     FILE_REGEX: re.Pattern = re.compile(r"^\w+ file:([0-9a-f]{64}) .*", re.IGNORECASE)
     DIGEST_REGEX: re.Pattern = re.compile(r"sha256:([0-9a-f]{64})$", re.IGNORECASE)
```

Comment:
It seems we're not using these anymore? Or at least not that I can see in the diff

Reply:
It's still in use.